services/service/server/server_linux.go

```diff
@@ -207,6 +229,7 @@ func (s *server) List(ctx context.Context, req *pb.ListRequest) (*pb.ListReply,
 
 // See: pb.ServiceServer.Status
 func (s *server) Status(ctx context.Context, req *pb.StatusRequest) (*pb.StatusReply, error) {
+	// req.DisplayTimestamp = true
```

Comment:
intentional?

Reply:
Or forget to delete, for initial testing purpose.